Workspaces/Base/Workspace.cs

```diff
@@ -82,12 +84,7 @@ public Bounds outerBounds
 		/// <summary>
 		/// If true, prevent the resizing of a workspace via the front and back resize handles
 		/// </summary>
-		public bool preventFrontBackResize { set { m_WorkspaceUI.preventFrontBackResize = value; } }
-
-		/// <summary>
-		/// If true, prevent the resizing of a workspace via the left and right resize handles
-		/// </summary>
-		public bool preventLeftRightResize { set { m_WorkspaceUI.preventLeftRightResize = value; } }
+		public bool preventResize { set { m_WorkspaceUI.preventResize = value; } }
```

Comment:
The purpose of the separate front/back left/right resize separation was to allow "2D" workspaces (profiler/console/etc) to only allow for the left/right resize handles to appear, but not the front/back handles.